app/src/main/res/values/themes.xml

```diff
@@ -2,15 +2,15 @@
     <!-- Base application theme. -->
     <style name="Theme.Move" parent="Theme.MaterialComponents.DayNight.NoActionBar">
         <!-- Primary brand color. -->
-        <item name="colorPrimary">@color/purple_500</item>
-        <item name="colorPrimaryVariant">@color/purple_700</item>
+        <item name="colorPrimary">@color/purple</item>
+        <item name="colorPrimaryVariant">@color/indigo</item>
         <item name="colorOnPrimary">@color/white</item>
         <!-- Secondary brand color. -->
-        <item name="colorSecondary">@color/teal_200</item>
-        <item name="colorSecondaryVariant">@color/teal_700</item>
+        <item name="colorSecondary">@color/pastel_purple</item>
+        <item name="colorSecondaryVariant">@color/pastel_pink</item>
         <item name="colorOnSecondary">@color/black</item>
         <!-- Status bar color. -->
-        <item name="android:statusBarColor" tools:targetApi="l">?attr/colorPrimaryVariant</item>
+        <item name="android:statusBarColor" >?attr/colorPrimary</item>
```

Comment:
This is a good solution 👍 
As an alternative, some prefer having a `status_bar_color` value instead of using attr, because some apps don't match it. And some might match it on light mode but not on dark mode.
So you would have
```
<item name="android:statusBarColor" >@color/status_bar_color</item>

in white theme colors:
<color name="status_bar_color">?attr/colorPrimary</color>

and in dark
<color name="status_bar_color">whateverelse</color>
```